backend/src/main/java/com/llm_service/llm_service/service/ConversationService.java

```diff
@@ -84,8 +78,9 @@ public void deleteAll() {
         conversationPersistenceManager.deleteAll();
     }
 
-    public Conversation edit(UUID id) {
-        return Conversation.builder().build();
+    public Conversation editTitle(UUID id, String title) {
+        return conversationPersistenceManager.save(
```

Comment:
i think `editTitle` should throw an error when a conversation does not exists , here it is just getting optional from the `getByID`

Reply:
it is handled in the controller